index.html

```diff
@@ -2120,588 +2138,378 @@ <h3>Thing Root Object</h3>
-in order to adopt existing (e.g., WebSub) or customer-oriented event mechanisms (e.g., Webhooks).
-For this reason, <code>subscription</code> and <code>cancellation</code> can be defined according to the desired mechanism. Please find further details in [[WoT-Binding-Templates]].
-Example <a href="#webhook-example-serialization"></a> illustrates how Events can use <code>subscription</code> and <code>cancellation</code> to describe Webhooks.
-</p>
+  Another possibility to set the default language is through a language negotiation mechanism,
+  such as the <code>Accept-Language</code> header field of HTTP.
+  <span class="rfc2119-assertion" id="td-ns-multilanguage-content-negotiation">
+    In cases where the default language has been negotiated,
+    an <code>@language</code> member MUST be present to indicate the result of the negotiation and the corresponding default language of the returned content.
+  </span>
+  <span class="rfc2119-assertion" id="td-ns-multilanguage-content-negotiation-no-multi">
+    When the default language has been negotiated successfully,
+    TD documents SHOULD include the appropriate matching values for the members <code>title</code> and <code>description</code>
+    in preference to <code>MultiLanguage</code> objects in <code>titles</code> and <code>descriptions</code> members.
+  </span>
+  <span class="rfc2119-assertion" id="td-ns-multilanguage-content-negotiation-optional">
+    Note however that Things MAY choose to not support such dynamically-generated TDs
+    nor to support language negotiation (e.g., because of resource constraints).
+  </span>
+  </p>
 
-</section> <!-- end of id="event-serialization-json"-->
+</section> <!-- end of id="human-readable-serialization-json"-->
 
-<section id="form-serialization-json">
-<h3><code>forms</code></h3>
+<section id="version-serialization-json">
+<h3><code>version</code></h3>
 
-  <p><span class="rfc2119-assertion" id="td-forms">
-    All name-value pairs of an instance of <code>Form</code>,
-    where the name is a <a>Vocabulary Term</a> included in the <a>Signature</a> of <code>Form</code>,
-    MUST be serialized as members of the JSON object that results from
-    serializing the <code>Form</code> instance, with the
-    <a>Vocabulary Term</a> as name.
+  <p><span class="rfc2119-assertion" id="td-version">
+    All name-value pairs of an instance of <code>VersionInfo</code>,
+    where the name is a <a>Vocabulary Term</a> included in the <a>Signature</a> of <code>VersionInfo</code>,
+    MUST be serialized as JSON members with the <a>Vocabulary Term</a> as name.
   </span></p>
 
-  <p><span class="rfc2119-assertion" id="td-form-protocolbindings">
-  If required, form objects
-  MAY be supplemented with protocol-specific <a>Vocabulary Terms</a>
-  identified with a prefix.
-  </span>
-  See also <a href="#protocol-bindings"></a>.
-  </p>
+  <p>The recommended version identification pattern value is to rely on the semantic versioning policy [[?SemVer]].</p>
```

Comment:
The grammar of this sentence is a little broken.   I think it should be "The recommended practice is to use a version identification pattern based on semantic versioning [[...]]".    It might be good to make this a SHOULD assertion, too: "The version SHOULD be identified with a value based on the semantic versioning pattern [[...]]"

Reply:
Added assertion